test/framework/deployments/kic/kubernetes.go

```diff
@@ -82,7 +82,7 @@ func (t *k8sDeployment) Deploy(cluster framework.Cluster) error {
 }
 
 func (t *k8sDeployment) Delete(cluster framework.Cluster) error {
-	return cluster.DeleteNamespace(t.ingressNamespace)
+	return cluster.(*framework.K8sCluster).TriggerDeleteNamespace(t.ingressNamespace)
```

Comment:
We use helm to install. Why don't we use helm to uninstall? Also I don't see why we delete a namespace that wasn't created in install...